### ISSUE TITLE
Site 075 - SAB is not saving updates and transactions

### ISSUE THREAD

**Issue by kdjstudios** [kdjstudios/SABillingGitlab#291]
In GitLab by @kdjstudios on Mar 30, 2017, 09:07

I was working on the Billing this evening.   I went back to an account that I created to ensure I didn’t miss anything.
I noticed that all the transactions were gone.   So, I entered them again, thinking I might not have saved the entries.   Updated the account and moved on. 
I went back for the second time to ensure this time, all the transactions are there.   This time only VCC patch was missing.  So, I entered and saved. This time I took a screenshot.
Saved and went back, it was missing.   So, I used the old patch transaction, saved.   
Again, when I logged back to the account, patch time was missing.  
 
![image006](/uploads/39652247009786fb32860160e22843cf/image006.png)![image007](/uploads/bc273adc2790e68aa93134d3fcdfe9ec/image007.png)I have to go back and double check my entries so far.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Mar 30, 2017, 09:31

@smasih @abakshi 

Faranak is stating that today is there billing day and they need to have this looked into as soon as possible.

**Comment by kdjstudios** [comment]
In GitLab by @smasih on Mar 31, 2017, 01:33

@kdjstudios  @abakshi 

as I checked, I was unable to replicate the issue.

I tried adding multiple activities and it stayed there until unless I removed it. 

Perhaps issue was with the network connection, however please request Frank to use  **'Save'**  button instead of **'save n next'** (in case if he used save n next), because 'save n next' is only to save and jump on next account to update VCC codes. 

In short,  I didn't find anything weird, but please if the problem still persist please let us know.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Mar 31, 2017, 11:57

@smasih @abakshi 

Thank you for the update. I have advised Faranak to try this again and will follow up with her response.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Apr 4, 2017, 08:46

@smasih @abakshi 

Followed up with Faranak a second time. If no response from her within a weeks time. We will close the ticket presuming it is working.

**Comment by kdjstudios** [comment]
In GitLab by @smasih on Apr 5, 2017, 02:08

@kdjstudios 

thanks for the update

**Comment by kdjstudios** [comment]
In GitLab by @pchaudhary on Jun 12, 2017, 03:57

@kdjstudios  
Can we close this?

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jun 12, 2017, 09:16

@pchaudhary 

Will be marking as completed.
